pagopa/pn-frontend
# payments api migration - pn-10936

## Short description

Migrated the payments api to bff ones. We have four api in total: download payment attachment api (one for PA and one for PF/PG), payment info api (one for PA/PF/PG) and payment cart api  (one for PA/PF/PG)

## List of changes proposed in this pull request

- Auto generated clients
- Removed useless code
- Updated redux layer
- Updated tests

## How to test

- login with PF and PG user -> go to notification detail -> check that the payment infos are correctly retrieved -> check also the cache mechanism
- login with PF and PG user -> go to notification detail -> download a payment (F24 and PagoPa)
- login with PF and PG user -> go to notification detail -> pay a PagoPa payment
- login with PA user -> go to notification detail -> download a payment (PagoPa)

TIPS: for pa use grossini, for pf use cristoforo (check for notifications with one payment, multi payments, pagoPa and f24 payments and paginated payments) and for pg use Dante (do the same checks done for pf)
